viewer/measures/lvp_coproxamol/definition.yaml

```diff
@@ -18,15 +18,15 @@ why_it_matters: |
 
     The average cost per item is **£488** (November 2024), which is an increase of **£223** since 2022.
```

Comment:
We should probably remove this. It's taken from OP but doesn't quite fit the secondary care data.
Or otherwise say "In primary care the average cost per item is..."